iam/README.md

```diff
@@ -37,7 +37,7 @@ Pass in optional validators for verifying specific scopes or other properties on
 
 ```go
 // myCustomValidator checks if the user can access deployment information.
-func myCustomValidator(p *ArmoryCloudPrincipal) error {
+func myCustomValidator(p *ArmoryCloudPrincipal) bs {
```

Comment:
I don't think `bs` is right?

Reply:
yeah IDEAs refactor with golang sometimes does some whacky stuff.